napi/src/js_values/mod.rs

```diff
@@ -257,7 +257,7 @@ macro_rules! impl_object_methods {
       where
         T: NapiValue,
       {
-        let key = CString::new(name)?;
+        let key = CString::new(name.to_owned())?;
```

Comment:
Don't know why this line is required.
If without `to_owned`, add `println!("anything")` between `let key....` and `check_status(...` could prevent `3221225477` exit too...